std/traits.d

```diff
@@ -1087,6 +1079,38 @@ template ParameterStorageClassTuple(func...)
     static assert(pstc[2] == STC.none);
 }
 
+/*****************
+ * Convert string tuple Attribs to ParameterStorageClass bits
+ * Params:
+ *      Attribs = string tuple
+ * Returns:
+ *      ParameterStorageClass bits
+ */
+ParameterStorageClass extractParameterStorageClassFlags(Attribs...)()
```

Comment:
transform this into `template extractParameterStorageClassFlags(Attribs...)`
```
enum extractParameterStorageClassFlags = () {
// the old function body
} ();

this will force the template to resolve to literal rather then a symbol.